exporters/prometheus/exporter.go

```diff
@@ -59,13 +59,13 @@ var _ metric.Reader = &Exporter{}
 type collector struct {
 	reader metric.Reader
 
-	disableTargetInfo    bool
-	withoutUnits         bool
-	targetInfo           prometheus.Metric
-	disableScopeInfo     bool
-	createTargetInfoOnce sync.Once
-	scopeInfos           map[instrumentation.Scope]prometheus.Metric
-	metricFamilies       map[string]*dto.MetricFamily
+	disableTargetInfo bool
+	withoutUnits      bool
+	targetInfo        prometheus.Metric
+	disableScopeInfo  bool
+	scopeInfos        map[instrumentation.Scope]prometheus.Metric
+	metricFamilies    map[string]*dto.MetricFamily
+	mu                sync.RWMutex
```

Comment:
Can you re-group the members of this struct such that the members protected by the mutex are together underneath the mutex and add a comment indicating as much?

Why a `RWMutex`?  I don't see anywhere that a read lock is taken.

Reply:
Fixed, thanks.